crates/turborepo-lib/src/shim.rs

```diff
@@ -256,10 +256,14 @@ impl TurboState {
             }
         };
 
-        formatcp!("turbo-{}-{}", OS, ARCH)
+        formatcp!("{}-{}", OS, ARCH)
     }
 
-    pub fn binary_name() -> &'static str {
+    pub const fn platform_package_name() -> &'static str {
```

Comment:
This was made const along with its parent since you can only call const functions in a const context.